packages/twenty-server/test/integration/search/unaccent-search.integration-spec.ts

```diff
@@ -0,0 +1,203 @@
+      const extensionCheck = await global.testDataSource.query(`
+        SELECT EXISTS(SELECT 1 FROM pg_extension WHERE extname = 'unaccent')
+      `);
+
+      expect(extensionCheck[0].exists).toBe(true);
+
+      // Check if unaccent_immutable function exists
+      const functionCheck = await global.testDataSource.query(`
+        SELECT EXISTS(
+          SELECT 1 FROM information_schema.routines
+          WHERE routine_name = 'unaccent_immutable'
+        )
+      `);
+
+      expect(functionCheck[0].exists).toBe(true);
+    });
+
+    it('should perform accent-insensitive search with real database', async () => {
+      // Create a temporary test table for this test
+      await global.testDataSource.query(`
+        CREATE TEMPORARY TABLE test_unaccent_search (
+          id SERIAL PRIMARY KEY,
+          content TEXT,
+          search_vector tsvector
+        )
+      `);
+
+      // Insert test data with accented characters
+      await global.testDataSource.query(`
+        INSERT INTO test_unaccent_search (content, search_vector) VALUES
+        ('café restaurant', to_tsvector('simple', public.unaccent_immutable('café restaurant'))),
+        ('naïve approach', to_tsvector('simple', public.unaccent_immutable('naïve approach'))),
+        ('résumé template', to_tsvector('simple', public.unaccent_immutable('résumé template'))),
+        ('regular text', to_tsvector('simple', public.unaccent_immutable('regular text')))
+      `);
+
+      // Test 1: Search for 'cafe' should find 'café restaurant'
+      const result1 = await global.testDataSource.query(`
+        SELECT content FROM test_unaccent_search
+        WHERE search_vector @@ to_tsquery('simple', public.unaccent_immutable('cafe'))
+      `);
+
+      expect(result1).toHaveLength(1);
+      expect(result1[0].content).toBe('café restaurant');
+
+      // Test 2: Search for 'naive' should find 'naïve approach'
+      const result2 = await global.testDataSource.query(`
+        SELECT content FROM test_unaccent_search
+        WHERE search_vector @@ to_tsquery('simple', public.unaccent_immutable('naive'))
+      `);
+
+      expect(result2).toHaveLength(1);
+      expect(result2[0].content).toBe('naïve approach');
+
+      // Test 3: Search for 'resume' should find 'résumé template'
+      const result3 = await global.testDataSource.query(`
+        SELECT content FROM test_unaccent_search
+        WHERE search_vector @@ to_tsquery('simple', public.unaccent_immutable('resume'))
+      `);
+
+      expect(result3).toHaveLength(1);
+      expect(result3[0].content).toBe('résumé template');
+    });
+
+    it('should perform accent-insensitive ILIKE search', async () => {
+      // Create temporary test table
+      await global.testDataSource.query(`
+        CREATE TEMPORARY TABLE test_unaccent_ilike (
+          id SERIAL PRIMARY KEY,
+          name TEXT
+        )
+      `);
+
+      // Insert test data
+      await global.testDataSource.query(`
```

Comment:
Theses tests are too low-level for integration tests. You could say it doesn't hurt but I'm worried merging this will set a bad precedent for what inegration tests are in the code base. 

As you see in the test I sent you, it usually call `makeGraphqlAPIRequest` so it's at a much higher level. You don't interact with the database but instead test and go through the graphql layer, etc

Did you try it and hit a roadblock?

Reply:
No problem at all. Your tests were interesting! Thanks for the great work!